scripts/deployment/deploy.sh

```diff
@@ -38,7 +17,32 @@ if [ -z ${COMMIT_ID} ]; then
     export COMMIT_ID="latest"
 fi
 
+if [ -z ${TRAVIS_BRANCH} ]; then
+    echo "Please set the TRAVIS_BRANCH first."
+fi
+
+env=${TRAVIS_BRANCH}
+
+if [[ ${env} == "production" ]]; then
+    HOSTNAME="evalai.cloudcv.org"
+elif [[ ${env} == "staging" ]]; then
+    HOSTNAME="evalai-staging.cloudcv.org"
+else
+    echo "Skipping deployment since commit not on staging or production branch."
+    exit 0
+fi
+
 case $opt in
+        auto_deploy)
```

Comment:
Can we please add docs for this command below - https://github.com/Cloud-CV/EvalAI/pull/2809/files#diff-2237c118ae9fb2de885c8df4c5c5f1d4R141